UserGuide.md

```diff
@@ -40,17 +40,20 @@
         * [Concise Example](#concise-example)
         * [Debugging Failed Runs / FAQ](#debugging-failed-runs-faq)
             * [Interrogating Cromwell](#interrogating-cromwell)
-            * [Changing a WDL didn’t change the error message from CARROT / Cromwell](#changing-a-wdl-didn’t-change-the-error-message-from-carrot-cromwell)
+            * [Failures in CARROT](#failures-in-carrot)
     * [Defining and Generating Reports](#defining-and-generating-reports)
         * [Report Definition Process](#report-definition-process)
             * [1. Prepare your Jupyter notebook](#1-prepare-your-jupyter-notebook)
-                * [1a. Python 3](#1a-python-3)
+                * [1a. Python 3 and R 4](#1a-python-3-and-r-4)
```

Comment:
I almost got up in arms about a "python 4"...

Reply:
You haven't switched to Python 4 yet?

Reply:
Python 4 is strongly-typed and comes with a package manager that is actually good and works.